src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -289,3 +289,9 @@
     (utils.string/contains-emoji? s)             :emoji
     (existing-account-names s)                   :existing-name
     (utils.string/contains-special-character? s) :special-character))
+
+(defn accounts-with-customization-color
+  [accounts]
+  (map (fn [{:keys [color] :as account}]
+         (assoc account :customization-color color))
+       accounts))
```

Comment:
Extracted this function as its used in the `sub` and also inside event handler code.

Reply:
Do we really want to add an extra key with the same information?

Consider using `clojure.set/rename-keys` if we don't need them twice

Reply:
I did not introduce this code, just reshuffled it. Anyways, I have removed this function now.